modules/gke-cluster-standard/variables.tf

```diff
@@ -85,6 +85,11 @@ variable "cluster_autoscaling" {
       min           = number
       max           = number
     })))
+    tpu_resources = optional(list(object({
+      resource_type = string
+      min           = number
+      max           = number
+    })))
```

Comment:
Why do we need to have separate structure for PUs? I think  we can use `gpu_resources` to set those

Reply:
I was also thinking about reusing the existing gpu_resources block but the name is not explicit and neither is our documentation that you can use the same block to configure TPU limits.

Would it be fine to rename this block to "accelerator_resources" and include in the documentation that this can include GPU or TPU? This would break existing use of the module though. 
Or just to update the README to say that you can configure TPU in the GPU block?

Reply:
We do not have any autoscaling example. Maybe you can add one, and say there, that `gpu_resources` supports both. Renaming this variable to `accelerator_resources` sound good too.